examples/rbd/template-csi-rbdplugin-provisioner-patch.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# This is a patch to the existing statefulset deployment of CSI rbdplugin.
+#
+# This is to be used when using `clusterID` instead of monitors or
+# monValueFromSecret in the StorageClass to specify the Ceph cluster to
+# provision storage from, AND when the value of `--configroot` option to the
+# CSI pods is NOT "k8s_objects".
+#
+# This patch file, patches in the specified secret for the 'clusterID' as a
+# volume, instead of the Ceph CSI plugin actively fetching and using kubernetes
+# secrets.
+#
+# NOTE: Update csi-rbdplugin DaemonSet as well with similar patch
+# Post substituting the <cluster-id> in all places execute,
+#   `kubectl patch statefulset csi-rbdplugin-provisioner --patch\
+#         "$(cat template-csi-rbdplugin-provisioner-patch.yaml)"`
+#   to patch the statefulset deployment.
+spec:
```

Comment:
the above template and this one look same to me, can't find any difference, are they different?

Reply:
Hmmm, true did not notice that. Will fold the instructions above into the one of the two files and remove the other.